scripts/stackstorm.js

```diff
@@ -66,6 +66,9 @@ env.ST2_SLACK_FAIL_COLOR = env.ST2_SLACK_FAIL_COLOR || 'danger';
 // Optional, if not provided, we infer it from the API URL
 env.ST2_AUTH_URL = env.ST2_AUTH_URL || null;
 
+// Optional, if not provided, we infer it from the API URL
+env.ST2_STREAM_URL = env.ST2_STREAM_URL || null;
```

Comment:
Let's include the corresponding description in `README.md`: https://github.com/StackStorm/hubot-stackstorm/blob/master/README.md#plugin-environment-variable-options